AC 2 - FINAL

Queiroz21 opened the pull request:
Grupo:
Thais gasparetto
Thiago Queiroz
João Victor Gongalvez
Vinicius Ribeiro
Vitor Antunes

JVGoncalves committed:
Ajustes em Agenda e Funções

JVGoncalves committed:
Ajuste de inserção

inserção na branch errada

thaisgasparetto committed:
Update funcoes.py

Correção de bugs

JVGoncalves committed:
Ajuste em Funções

inserção na branch errada

thaisgasparetto committed:
Update agenda.py

Correção de bugs

JVGoncalves committed:
Alteração em agenda e funções

Queiroz21 committed:
Atualização e Criação da Busca

Criação da função de busca
Alteração da função bemvindo

thaisgasparetto committed:
Update funcoes.py

Correção de bugs

thaisgasparetto committed:
Update agenda.py

Correção de bugs

Queiroz21 committed:
Merge branch 'master' into thaisgasparetto

Queiroz21 committed:
Merge pull request #3 from Queiroz21/thaisgasparetto

Thaisgasparetto para Master

Queiroz21 committed:
Merge branch 'master' into Queiroz21

Queiroz21 committed:
Merge pull request #2 from Queiroz21/Queiroz21

Atualização e Criação da Busca
Queiroz para Master

Queiroz21 committed:
Merge branch 'master' into jvgoncalves

Queiroz21 committed:
Merge pull request #1 from Queiroz21/jvgoncalves

Alteração em agenda e funções
inserção da função sair do programa

V1T0RAB committed:
Atualização de funções

A atualização da função listar para exibição de todo o arquivo

Queiroz21 committed:
Merge pull request #4 from Queiroz21/V1T0RAB

Atualização de funções

umvini committed:
Criação da função de deletar contato

Queiroz21 committed:
Atualização do Script e reinserção dE FUNÇÕES

Métodos Busca e SairDoPrograma foram inseridos novamente após terem sido deletados com a atualização última atualização

Queiroz21 committed:
Merge branch 'master' into viniciusr258

Queiroz21 committed:
Merge pull request #5 from Queiroz21/viniciusr258

Criação da função de deletar contato

Queiroz21 committed:
Ajuste na estrutura de controle

Inserção das novas funções

Queiroz21 committed:
Atualização metodo BemVindo

Inserção das ovas funções

Queiroz21 committed:
Atualização da estrutura de controle

ajuste do valor para cada função ser chamada

thaisgasparetto committed:
Add files via upload

Mais correções de bugs

Queiroz21 committed:
Merge branch 'master' into thaisgasparetto

Queiroz21 committed:
Merge pull request #6 from Queiroz21/thaisgasparetto

Atualização de Bugs V2

Queiroz21 committed:
Inserção de método de controle

Adicionada função de controle no Arquivo funções, e remoção dos if e else do arquivo agenda, tornando tudo aquilo uma função

Queiroz21 committed:
Merge branch 'master' into Queiroz21

Queiroz21 committed:
Merge pull request #7 from Queiroz21/Queiroz21

Inserção de método de controle

umvini committed:
Atualização função de deletar

umvini committed:
indentação do codigo de deletar

Queiroz21 committed:
Merge pull request #8 from Queiroz21/viniciusr258

ajuste da função erro

Queiroz21 committed:
Add files via upload

Ajustes de sintaxe e detalhes da inserção e exclusão

Queiroz21 committed:
Merge branch 'master' into Queiroz21

Queiroz21 committed:
Merge pull request #9 from Queiroz21/Queiroz21

Ajuste de sintaxe na inserção e exclusão

JVGoncalves committed:
Adicionando apontador do travis

JVGoncalves committed:
Atualização de função

remoção de interação dentro das funções

rseteves commented:
Bug corrigido: OK
Loop: OK
Adicionar: OK 
Listar: OK
Buscar: Nao
Deletar: Ok

Nota: 8.5

Queiroz21 committed:
Atualização de funções e criação de teste unitario

Funções com ajustes das funções para adição do try catch para ajuste dos testes
Criação dos testes unitários de Cadastra, Lista e Deleta contatos

Queiroz21 committed:
Rename travis.yml to .travis.yml

Queiroz21 committed:
Update README.md
